### PR TITLE
Allow to disable writing computation times

### PR DESCRIPTION
**Pull request by bmwiedemann**
Allow to disable writing computation times to allow for reproducible builds.

Default to not write durations. if SOURCE_DATE_EPOCH indicates that a reproducible build is wanted.

Helps with https://github.com/pygraphviz/pygraphviz/issues/541

### CODE REVIEW

**Comment by larsoner** [comment]
Makes sense to me, can you add it to the [configuration doc](https://github.com/sphinx-gallery/sphinx-gallery/blob/master/doc/configuration.rst) somewhere?

**Comment by bmwiedemann** [comment]
I added some docs. Feel free to adjust it as you like.

File: doc/configuration.rst [hunk 1]
```diff
@@ -2293,6 +2294,15 @@ The ``min_reported_time`` parameter can be set to a number of seconds. The
 duration of scripts that ran faster than that amount will not be logged nor
 embedded in the html output.
 
+.. _write_computation_times:
+
+Write computation times
+=======================
+
+Set to ``false`` if you want to omit computation times from the output.
```
Comment (drammock) [inline doc/configuration.rst:2302]:
```suggestion
Set to ``False`` if you want to omit computation times from the output.
```

File: doc/configuration.rst [hunk 2]
```diff
@@ -2293,6 +2294,15 @@ The ``min_reported_time`` parameter can be set to a number of seconds. The
...
+This helps with reproducible builds.
+If the ``SOURCE_DATE_EPOCH`` environment variable is set, it defaults to false.
+True otherwise.
```
Comment (drammock) [inline doc/configuration.rst:2305]:
```suggestion
Default is ``True`` unless the ``SOURCE_DATE_EPOCH`` environment variable is set.
```

File: doc/configuration.rst (hunk 1 above)
Comment (lucyleeow) [inline doc/configuration.rst:2302]:
```suggestion
Set to ``False`` if you want to omit computation times from all gallery outputs.
```

**Review by lucyleeow — APPROVED** [review]
LGTM

**Pull request merged**